Curs/src/events/newEvaluationEvents.js

```diff
@@ -0,0 +1,79 @@
+interviewApp.newEvaluationEvents = function () {
+
+        const myObj = {}
+    
+    const goToTheEvaluationPage = function(event) {
+        event.preventDefault();
+
+        interviewApp.evaluationsModule.init() // load evaluation page
+
+    }
+
+    // get evaluationButton by his ID
+    const evaluationButton = document.getElementById("evaluation-page")
+      
+    // add "click" event listener
+    evaluationButton.addEventListener("click", goToTheEvaluationPage)
+        
+    // get button by his ID #submit-button
+    const submitFormButton = document.getElementById("submit-button")
+
+        // Create submitButton function
+        const submitButton = function(event) {
+            event.preventDefault();
+
+            const getSelectorAll = document.querySelectorAll("select")
+      
+            getSelectorAll.forEach(function(el) {
+                const nameSelector = el.name
+                const valueSelector = el.value  
+
+                myObj[nameSelector] = valueSelector
+
+            })
+
+            const getTextareaAll = document.querySelectorAll("textarea")
+
+            getTextareaAll.forEach(function(el) {
+                const nameTextarea = el.name
+                const valueTextarea = el.value
+
+                myObj[nameTextarea] = valueTextarea
+
+            })
+
+            const getInputAll = document.querySelectorAll("input")
+            getInputAll.forEach(function(el) {
+                const nameInput = el.name
+                const valueInput = el.value
+
+                if(el.type === 'text' || el.type === 'date') { // check the input type
+                    myObj[nameInput] = valueInput
+                } else if (el.type === 'radio' && el.checked) { // check input type and if the input is checked
+                    myObj[nameInput] = valueInput
+                }
+            })
+            console.log(myObj) // display the Object
```

Comment:
As a general rule, `console.log` / `debugger` and similar debugging-only instructions should be cleaned before they ever hit the codebase.

Reply:
👍